src/packages.js

```diff
@@ -452,6 +463,22 @@ export default class Packages {
     });
   }
 
+  /**
+   * Preloads background files of a set of packages
+   * @param {PackageMetadata[]} list Package list
+   */
+  _preloadBackgroundFiles(list) {
+    const backgroundFiles = list.reduce((filenames, iter) => [
+      ...filenames,
+      ...this._getPreloads(iter, 'background', 'apps')
```

Comment:
@andersevenrud I changed this line. It was like this:
```js
...filenames,
...metadataFilesToFilenames(
  filterMetadataFilesByType(metadata.files, fileType)
)
```

Reply:
Alrighty.